pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -1050,22 +1050,43 @@ public void validateTopicOperation(TopicName topicName, TopicOperation operation
     }
 
     public void validateTopicOperation(TopicName topicName, TopicOperation operation, String subscription) {
+        try {
+            validateTopicOperationAsync(topicName, operation, subscription).get();
+        } catch (InterruptedException | ExecutionException e) {
+            Throwable cause = e.getCause();
+            log.error("Validate Topic [%s] operation [%s] got server internal error.", cause);
```

Comment:
Please correct the log content.

Reply:
Removed it. :)
Since we're throwing a new exception below, I don't think we need to log here anymore.